tests/acceptance/features/apiSharingNgShareInvitation/shareInvitations.feature

```diff
@@ -3182,3 +3182,70 @@ Feature: Send a sharing invitations
       | permissionsRole | Viewer       |
     Then the HTTP status code should be "200"
     And user "Brian" should have a share "textfile.txt" shared by user "Alice" from space "NewSpace"
+
+  @env-config
+  Scenario Outline: list all drives after sharing a resource with denied permission role
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has uploaded file with content "personal space" to "lorem.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | <resource> |
+      | space           | Personal   |
+      | sharee          | Brian      |
+      | shareType       | user       |
+      | permissionsRole | Denied     |
+    When user "Brian" lists all spaces via the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON response should contain space called "Brian Murphy" and match
```

Comment:
you should check whether the share exists or not. checking personal drive is out of the scope of this scenario

Reply:
sorry for being vague in the ticket. I have updated the point https://github.com/owncloud/ocis/issues/10656